rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1578,19 +1578,32 @@ public void addSignature(BtcECKey federatorBtcPublicKey, List<byte[]> signatures
 
         Context.propagate(btcContext);
 
+        if (svpIsOngoing()) {
+            provider.getSvpSpendTxWaitingForSignatures()
+                .filter(svpSpendTxWFS -> svpSpendTxWFS.getKey().equals(rskTxHash))
```

Comment:
It's not a big deal, but I will also rename it `svpSpendTxWFS` to `svpSpendTxWFSEntry`.

Reply:
i dont think it adds value